relay-metrics/src/aggregation.rs

```diff
@@ -651,9 +651,10 @@ impl Error for ParseBucketError {
 /// To parse a submission payload, use [`Bucket::parse_all`].
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Bucket {
-    /// The start time of the time window. The length of the time window is stored in the
-    /// [`Aggregator`].
+    /// The start time of the time window.
     pub timestamp: UnixTimestamp,
+    /// The length of the time window in seconds.
+    pub width: u64,
```

Comment:
A `u32` would give us 136 years, but I decided to stick with `u64` to reduce the number of conversions, and because
* there's no difference in size for the serialized JSON,
* `Bucket` objects are not stored in the aggregator (it stores `BucketValue` objects).